pyproject.toml

```diff
@@ -44,3 +142,30 @@ select = ["ALL"]
 
 [tool.ruff.lint.pylint]
 max-args = 10
+
+[tool.black]
+exclude = "3rdparty/*"
+line-length = 100
+
+[tool.isort]
+profile = "black"
+src_paths = ["python"]
+
+[tool.cibuildwheel]
+build-verbosity = 1
+skip = ["cp36-*", "cp37-*", "cp38-*", "pp*", "*musllinux*", "cp313-manylinux_aarch64", "cp313-macosx_x86_64"] # pypy doesn't play nice with pybind11
```

Comment:
Why skip cp313-manylinux_aarch64 and cp313-macosx_x86_64?

Reply:
python 3.13 on aarch64 and macosx x86 don't have torch versions for torch 2.5.0+. macos x64 is likely never going to never see any new torch builds for any python version after 3.13. The missing Linux aarch64 build for python 3.13 is mostly because python 3.13 is still just in early support for torch. I'll add a comment for the future.